packages/node_modules/@webex/internal-plugin-calendar/test/integration/spec/calendar.js

```diff
@@ -90,9 +90,53 @@ function postToWhistler(webex, type, payload) {
     }));
 }
 
-describe.skip('plugin-calendar', () => {
+function postNotesToWhistler(webex, type, payload) {
+  return retry(() => webex.request({
+    method: 'POST',
+    service: 'whistler',
+    resource: 'calendarEvent',
+    body: payload,
+    qs: {
+      changeType: type,
+      useProduction: typeof process.env.WDM_SERVICE_URL === 'undefined' || process.env.WDM_SERVICE_URL.includes('wdm-a.wbx2.com')
+    }
+  })
+    .then((res) => {
+      // Test response is using our parameters
+      const createdMeeting = res.body;
+
+      assert.isDefined(createdMeeting);
```

Comment:
Please move these assertions into a test instead of the helper function. A failed assertion here would be difficult to track down due to it being embedded in other tests.

For example, if the object returned didn't have the meeting notes, we would get a failure on the test `'retrieves the meeting notes for the given meetingId'` but the failure would be on the request, not the implementation.

Reply:
I was using the existing code in that file as reference, I'll move it elsewhere or remove it.